templates/mainapp/layout.html

```diff
@@ -39,7 +39,7 @@
     <header>
         <div id="topheader">
             <nav class="navbar navbar-expand-md navbar-light">
-                <a class="navbar-brand head-text" href="/"><img width="150px" src="https://user-images.githubusercontent.com/45713796/98271308-d18aac80-1fb5-11eb-9db3-dda942cc1b07.png"></a>
+                <a class="navbar-brand head-text" href="/"><img width="200px" src="./static/mainapp/png/brand-img.png"></a>
```

Comment:
Change the src to [this way](https://github.com/Praful932/Kitabe/blob/91c168d1bcd5888c835fc48bf3ff3e0ac05064b7/templates/mainapp/index.html#L19)  it is the way to specify the path of the static way in Django template.